packages/terra-avatar/src/terra-dev-site/doc/avatar/Avatar.2.doc.mdx

```diff
@@ -0,0 +1,47 @@
+import { Badge } from '../../../../package.json?dev-site-package';
+
+import AvatarUser from '../example/avatar/AvatarUser?dev-site-example';
+import AvatarOneInitial from '../example/avatar/AvatarOneInitial?dev-site-example';
+import AvatarTwoInitials from '../example/avatar/AvatarTwoInitials?dev-site-example';
+import AvatarImage from '../example/avatar/AvatarImage?dev-site-example';
+import AvatarIsDeceased from '../example/avatar/AvatarIsDeceased?dev-site-example';
+import AvatarSize from '../example/avatar/AvatarSize?dev-site-example';
+import AvatarColorVariants from '../example/avatar/AvatarColorVariants?dev-site-example';
+
+import AvatarPropsTable from '../../../../src/variants/Avatar?dev-site-props-table';
+
+<Badge />
+
+# Terra Avatar
```

Comment:
It looks like we copied over the content from `docs/avatar.md/`. Do we need to delete `'../../../../docs/avatar.md';`?

Reply:
updated here https://github.com/cerner/terra-core/pull/2810/commits/14dc3d156f17f1c1a07ad277088e105d5a9c9fe6